src/pages/how-to/routing-traffic-to-private-networks.mdx

```diff
@@ -49,15 +51,49 @@ Nodes connected to routing peers will choose one of them to route packets to ext
 Masquerade hides other NetBird network IPs behind the routing peer local address when accessing the target Network range. This option allows access to your private networks without configuring routes on your local routers or other devices.
 
 If you don't enable this option, you must configure a route to your NetBird network in your external network infrastructure.
+
+### DNS Routes
+An alternative to specifying a network range directly is to use DNS routes. Instead of adding the network directly, you can add multiple domains in a route that will be dynamically resolved on the client. The resolved IP addresses for these domains will be added as routes.
```

Comment:
```suggestion
An alternative to specifying a network range directly is to use DNS routes. Instead of adding the network directly, you can add multiple domains in a route that will be dynamically resolved on the client. The resolved IP addresses for these domains will be added as routes. For example, a network administrator can ensure that traffic to `google.com` or `api.website.com` is routed through a specific machine. So they configure DNS routes for these domains instead of specifying the IP ranges.
```

Reply:
I think `google.com` is not the best example as it redirects to `www.google.com` and calls a bunch of subdomains from there

Reply:
lets use website.com :)